examples/torch/README.md

```diff
@@ -0,0 +1,22 @@
+Install the packages needed for samples by running the following in the current directory:
+
+```
+pip install -r requirements.txt
+```
+
+One of the needed package - torchvision.
+The minor version of torchvision should always match the minor version of installed torch package.
+torchvision minor version is torch minor version +1. For instance, torch 1.8.2 goes with torchvision 0.9.2.
```

Comment:
What do you think about adding a link to the torchvision torchvision version compatibility table instead of giving the recipe? For example: https://github.com/pytorch/pytorch/wiki/PyTorch-Versions or https://pypi.org/project/torchvision/

Reply:
That's a good idea! Will add it in the next PR as it doesn't affect code and doesn't require one more validation cycle.